app/src/main/java/com/kickstarter/ui/adapters/DiscoveryPagerAdapter.kt

```diff
@@ -48,52 +48,56 @@ class DiscoveryPagerAdapter(
      * Passes along root categories to its fragment position to help fetch appropriate projects.
      */
     fun takeCategoriesForPosition(categories: List<Category>, position: Int) {
-        Observable.from(fragments)
+        Observable.fromIterable(fragments)
```

Comment:
TODO: Quite sure we can get rid in the adapter of all the RXJava subscriptions. Seems the only goal of having an observable from a collection and operate on it instead of directly operating on the collection (by not having it be mutable) is to avoid concurrency issues.  
Should create a ticket on our techDebt backlog to addres it outside the RXJava migration